HLTrigger/Egamma/plugins/HLTPMMassFilter.cc

```diff
@@ -20,6 +20,10 @@ HLTPMMassFilter::HLTPMMassFilter(const edm::ParameterSet& iConfig) : HLTFilter(i
 
   lowerMassCut_ = iConfig.getParameter<double>("lowerMassCut");
   upperMassCut_ = iConfig.getParameter<double>("upperMassCut");
+  lowerdRCut_ = iConfig.getParameter<double>("lowerdRCut");
+  upperdRCut_ = iConfig.getParameter<double>("upperdRCut");
+  lowerdR2Cut_ = lowerdRCut_ * lowerdRCut_;
+  upperdR2Cut_ = upperdRCut_ * upperdRCut_;
```

Comment:
```suggestion
  lowerdR2Cut_ = lowerdRCut_ >=0 ? lowerdRCut_ * lowerdRCut_ : 0;
  upperdR2Cut_ = upperdRCut_ >=0 ? upperdRCut_ * upperdRCut_ : 99999;
```
Please don't take this code suggestion literally, it's not particularly elegant.

I'm just worried that a future user will use a negative value assuming that this means "no cut". Do you think we should guard against that ?

Reply:
FWIW, I think it's a good suggestion and I support applying it.